.github/workflows/ci.yaml

```diff
@@ -49,15 +45,14 @@ jobs:
         sudo cp lib/*.a /usr/lib
         sudo ln -s /usr/lib/libgtest.a /usr/local/lib/libgtest.a
         sudo ln -s /usr/lib/libgtest_main.a /usr/local/lib/libgtest_main.a
-
     - name: Clone
       uses: actions/checkout@v3
-      with: 
+      with:
         submodules: true
     - name: Test
       run: |
         . ~/.spack/Spack/share/spack/setup-env.sh
-        export Kokkos_DIR=/home/runner/.spack/Spack/opt/spack/linux-ubuntu22.04-x86_64_v4/gcc-12.1.0/kokkos-3.7.01-zhffzchmugzhigeekgnatfeikkrjwmfw
+        export Kokkos_DIR=$(spack find -p kokkos | grep -m1 kokkos | awk '{print $2}')
```

Comment:
@faisal-bhuiyan thanks, I think this is good! Please explain the sections of this command either in a comment in the yaml file or in the pull request description for the benefit of future developers.

Reply:
@rafmudaf Please find the requested explanation added to both places you mentioned. Let me know if there is anything else we need to do here.